config/core/deployments/activator.yaml

```diff
@@ -87,14 +87,20 @@ spec:
         - name: h2c
           containerPort: 8013
 
-        readinessProbe: &probe
+        readinessProbe:
           httpGet:
             port: 8012
             httpHeaders:
             - name: k-kubelet-probe
               value: "activator"
           failureThreshold: 12
-        livenessProbe: *probe
+        livenessProbe:
+          httpGet:
+            port: 8012
+            httpHeaders:
+            - name: k-kubelet-probe
+              value: "activator"
+          failureThreshold: 18
```

Comment:
@julz how much bigger do you think this should be comparing to readinessProbe?

Reply:
have to admit, I don't know - other than that it should be more that readiness threshold, which this is :) 

cc @vagababov for thoughts on how long indicates deadlock rather than overload in activator

Reply:
In my experience this usually indicates that Activator cannot speak to the Autoscaler. I am not sure if we haven't become ready after 12 we need to wait more.

Reply:
Thanks for the comments! In this case, another (18 - 12) * 10 = 1 min is probably too much, so I just reverted `failureThreshold` back to 12 times, and added 15 `initialDelaySeconds` for liveness instead. Is this okay? Or we prefer to keep them using the same probe?